api/src/main/java/org/openmrs/module/pihcore/setup/MetadataSharingSetup.java

```diff
@@ -0,0 +1,150 @@
+package org.openmrs.module.pihcore.setup;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.metadatasharing.ImportConfig;
+import org.openmrs.module.metadatasharing.ImportMode;
+import org.openmrs.module.metadatasharing.ImportedPackage;
+import org.openmrs.module.metadatasharing.MetadataSharing;
+import org.openmrs.module.metadatasharing.api.MetadataSharingService;
+import org.openmrs.module.metadatasharing.wrapper.PackageImporter;
+import org.openmrs.module.pihcore.PihCoreUtil;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Iterator;
+import java.util.List;
+
+public class MetadataSharingSetup {
+
+    protected static final Log log = LogFactory.getLog(MetadataSharingSetup.class);
+
+    public static void installMetadataSharingPackages() {
+
+        try {
+            Collection<File> mdsFiles = loadMdsFiles();
+
+            if (mdsFiles == null) {
+                log.error("Unable to find any metadata sharing files");
```

Comment:
This is redundant with the error messages in `loadMdsFiles`

Reply:
Good point, removed.